lib/chord-metadata/docker-compose.yml

```diff
@@ -41,6 +41,7 @@ services:
       - CANDIG_AUTHORIZATION=${CANDIG_AUTHORIZATION}
       - CACHE_TIME=${CACHE_TIME}
       - CANDIG_OPA_SECRET=${CANDIG_OPA_SECRET}
+      - CANDIG_OPA_SITE_ADMIN_KEY=${OPA_SITE_ADMIN_KEY}
```

Comment:
Avoid using secrets in environment variables if possible

Reply:
This one isn't a secret: it's something like "site-admin"